pennylane/_qubit_device.py

```diff
@@ -121,7 +125,7 @@ def _scatter(indices, array, new_dimensions):
         "Projector",
     }
 
-    def __init__(self, wires=1, shots=None, cache=0, analytic=None):
+    def __init__(self, wires=1, shots=None, cache=0, analytic=None, custom_decomps=None):
```

Comment:
@mlxd just thought I'd tag you because we are adding stuff to the `QubitDevice` class that may have to be ported to `Device` one day

Reply:
Great, thanks for the heads-up @mariaschuld !